cluster-autoscaler/utils/drain/drain_test.go

```diff
@@ -476,75 +476,74 @@ func TestDrain(t *testing.T) {
 			description: "Job-managed pod with PodSafeToEvict=false annotation",
 			pods:        []*apiv1.Pod{unsafeJobPod},
 			pdbs:        []*policyv1.PodDisruptionBudget{},
-			rcs:         []apiv1.ReplicationController{rc},
+			rcs:         []*apiv1.ReplicationController{&rc},
 			expectFatal: true,
 			expectPods:  []*apiv1.Pod{},
 		},
 		{
 			description: "empty PDB with RC-managed pod",
 			pods:        []*apiv1.Pod{rcPod},
 			pdbs:        []*policyv1.PodDisruptionBudget{emptyPDB},
-			rcs:         []apiv1.ReplicationController{rc},
+			rcs:         []*apiv1.ReplicationController{&rc},
 			expectFatal: false,
 			expectPods:  []*apiv1.Pod{rcPod},
 		},
 		{
 			description: "kube-system PDB with matching kube-system pod",
 			pods:        []*apiv1.Pod{kubeSystemRcPod},
 			pdbs:        []*policyv1.PodDisruptionBudget{kubeSystemPDB},
-			rcs:         []apiv1.ReplicationController{rc},
+			rcs:         []*apiv1.ReplicationController{&kubeSystemRc},
 			expectFatal: false,
 			expectPods:  []*apiv1.Pod{kubeSystemRcPod},
 		},
 		{
 			description: "kube-system PDB with non-matching kube-system pod",
 			pods:        []*apiv1.Pod{kubeSystemRcPod},
 			pdbs:        []*policyv1.PodDisruptionBudget{kubeSystemFakePDB},
-			rcs:         []apiv1.ReplicationController{rc},
+			rcs:         []*apiv1.ReplicationController{&kubeSystemRc},
 			expectFatal: true,
 			expectPods:  []*apiv1.Pod{},
 		},
 		{
 			description: "kube-system PDB with default namespace pod",
 			pods:        []*apiv1.Pod{rcPod},
 			pdbs:        []*policyv1.PodDisruptionBudget{kubeSystemPDB},
-			rcs:         []apiv1.ReplicationController{rc},
+			rcs:         []*apiv1.ReplicationController{&rc},
 			expectFatal: false,
 			expectPods:  []*apiv1.Pod{rcPod},
 		},
 		{
 			description: "default namespace PDB with matching labels kube-system pod",
 			pods:        []*apiv1.Pod{kubeSystemRcPod},
 			pdbs:        []*policyv1.PodDisruptionBudget{defaultNamespacePDB},
-			rcs:         []apiv1.ReplicationController{rc},
+			rcs:         []*apiv1.ReplicationController{&kubeSystemRc},
 			expectFatal: true,
 			expectPods:  []*apiv1.Pod{},
 		},
 	}
 
 	for _, test := range tests {
-		fakeClient := &fake.Clientset{}
-		register := func(resource string, obj runtime.Object, meta metav1.ObjectMeta) {
-			fakeClient.Fake.AddReactor("get", resource, func(action core.Action) (bool, runtime.Object, error) {
-				getAction := action.(core.GetAction)
-				if getAction.GetName() == meta.GetName() && getAction.GetNamespace() == meta.GetNamespace() {
-					return true, obj, nil
-				}
-				return false, nil, fmt.Errorf("Not found")
-			})
-		}
+		var err error
+		var rcLister v1lister.ReplicationControllerLister
```

Comment:
I would prefer `replicationControllerLister` as variable name. Abbreviations are bad!!!!

Reply:
Why? I bet go naming convention likes them (it seems to hate long variable names).

Reply:
This does not mean thay are good.
As long as longer variable name does not make lines very long they there is no point in using abbreviations.
They are just harder to read and makes understanding the code harder. Unless you are reading letter by letter of course....